composer.json

```diff
@@ -53,7 +53,7 @@
         "doctrine/dbal":                        "^2.10",
         "doctrine/doctrine-module":             "^4.0",
         "doctrine/orm":                         "^2.6.4",
-        "doctrine/persistence":                 "^1.3",
+        "doctrine/persistence":                 "^1.3.3||^2.0",
```

Comment:
do we really have to support v1 doctrine/persistance? https://github.com/doctrine/DoctrineModule/blob/4.1.x/composer.json#L54 or this for older doctrine-module versions? If not maybe just require `doctrine/doctrine-module:^4.1`

Reply:
Though I think requiring `doctrine/doctrine-module:^4.1` would be reasonable I am unsure if this should be done in an `3.1.x` release. Loosing compatibility to `doctrine/persistence` within a bugfix release seems to break semantic versioning. So from my unserstanding you'd have to go to `3.2.x` when upgrading from `doctrine/doctrine-module:^4.0` to `^4.1`. Therefore, it might be better to keep it with `^1.3 || ^2.0` for now?

Reply:
v1 dropped in https://github.com/doctrine/DoctrineORMModule/pull/643/commits/ed2bd4364b3df3a6ee4ab8490a2d1ffd4b07aea4

Reply:
Agreed, driehle.  This PR is not the place to limit the persistence version to 2.0 only.

Reply:
this whole PR is not a bugfix and will break semantic versioning, then we can release 3.2.0 no?